src/stirling/source_connectors/socket_tracer/testing/containers/ssl/nginx.conf

```diff
@@ -11,7 +11,7 @@ http{
     ssl_certificate /etc/ssl/server.crt;
     ssl_certificate_key /etc/ssl/server.key;
 
-    ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
+    ssl_protocols TLSv1 TLSv1.1 TLSv1.2 TLSv1.3;
```

Comment:
Without this, the Nginx container will fail to negotiate a TLS 1.3 connection. This is triggered in the trace bpf test.